Orleans/src/Microsoft.Orleans.Host.Abstractions/StorageProvidersFeature.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Orleans.Host.Abstractions
+{
+    // this would go into the (optional) storage providers DLL.
+    public interface IStorageProvidersFactory
```

Comment:
Yup, I agree, remnants of previous prototyping. This is more an IStorageProviderManager